### PR TITLE
Resolving misleading logs

### PR DESCRIPTION
**Pull request by tPeeush**

### CODE REVIEW

File: protocol/src/main/java/org/fidoalliance/fdo/protocol/HttpClient.java [hunk 1]
```diff
@@ -275,6 +275,8 @@ public void run() {
         } catch (Throwable e) {
           logger.error("failed to send error");
         }
+      } else {
+        throw new RuntimeException();
```
Comment (DukeDavis12) [inline protocol/src/main/java/org/fidoalliance/fdo/protocol/HttpClient.java:279]: Where is the Exception getting caught?

Comment (tPeeush) [reply protocol/src/main/java/org/fidoalliance/fdo/protocol/HttpClient.java:279]: Line no. 77 in DeviceApp.java

File: component-samples/device/src/main/java/org/fidoalliance/fdo/sample/DeviceApp.java [hunk 2]
```diff
@@ -84,7 +84,7 @@ public void run() {
       } catch (IOException e) {
         logger.error("failed log exception");
         // already in exception handler
-      }
+      } throw new RuntimeException();
```
Comment (DukeDavis12) [inline component-samples/device/src/main/java/org/fidoalliance/fdo/sample/DeviceApp.java:87]: Why are we throwing exception after catch ?

File: component-samples/device/src/main/java/org/fidoalliance/fdo/sample/DeviceApp.java [hunk 3]
```diff
@@ -72,11 +72,12 @@ public void run() {
     try {
       logger.info("Starting Fdo Device");
       super.run();
-      logger.info("Starting Fdo Completed");
+      logger.info("Fdo Completed");
```
Comment (DukeDavis12) [inline component-samples/device/src/main/java/org/fidoalliance/fdo/sample/DeviceApp.java:75]:
```suggestion
      logger.info("FDO Completed");
```

Comment (v-vasavi) [inline component-samples/device/src/main/java/org/fidoalliance/fdo/sample/DeviceApp.java:75]: Is this FDO completed or FDO Device Initialization completed?

**Pull request closed without merging**